src/NosCore.Packets/Enumerations/QuestType.cs

```diff
@@ -38,5 +34,6 @@ public enum QuestType : short
         Inspect = 24,
         WinRaid = 25,
         FlowerQuest = 26,
+        Battlepass = 27
```

Comment:
that should not be here it's not on the QuestType from official it need to be mapped like the equipments